cockpit/webpack.config.ts

```diff
@@ -0,0 +1,71 @@
+const path = require('path');
```

Comment:
The `isOnPrem` env var  is constant during the entire a, so we can leverage webpack to build the relevant modules, this also ensure that the environment-specific modules are code-split and loaded only when necessary

i.e
```js
module.exports = {
// ...
  plugins: [
    new webpack.DefinePlugin({
      // This replaces `process.env.IS_ON_PREMISE` with `true` or `false` at build time
      'process.env.IS_ON_PREMISE': JSON.stringify(isOnPremise),
    }),
    // ... other plugins
  ],
  resolve: {
    alias: {
      // Use different modules based on the environment
      // For example, `api` will resolve to `cloudApi` or `imageBuilderApi` accordingly
      api: isOnPremise
        ? path.resolve(__dirname, 'src/store/imageBuilderApi.ts')
        : path.resolve(__dirname, 'src/store/cloudApi.ts'),
    },
  },
```